CheckboxItem/CheckboxItem.module.less

```diff
@@ -7,16 +7,19 @@
 	align-items: center;
 
 	.applySkins({
-		padding-left: @agate-checkboxitem-padding-left;
-		padding-right: @agate-checkboxitem-padding-right;
-		border-color: @agate-checkboxitem-border-color;
+        height: @agate-checkboxitem-height;
 		font-size: @agate-checkboxitem-font-size;
 
 		&.selected {
 			border-color: @agate-checkboxitem-selected-border-color;
+
 			&::before {
 				content: none; // Overwrite the selected styles for Item
 			}
+
+          .focus({
+            border-color: @agate-checkboxitem-focus-border-color;
+          });
```

Comment:
I'm not sure this is needed. I think we should keep consistency with Item. I will also ask the designer to correct the GUI to specifically show that the border is white with 40% opacity
![image](https://user-images.githubusercontent.com/63335068/114143521-f09eb900-991c-11eb-84a2-2951073c1d07.png)

Reply:
But what about this specific requirement?
![image](https://user-images.githubusercontent.com/63341832/114144169-b550ba00-991d-11eb-92a6-bba617489129.png)

Reply:
As I said, I'm not sure. This is not a blocker comment

Reply:
I think we should leave this as is until we get an answer for this issue

Reply:
also please check indentation